### PR TITLE
add DreamMaker_OverLord print info

### PR DESCRIPTION
**Pull request by leyap**

### CODE REVIEW

File: spark-print-data/printertypes.json [hunk 1]
```diff
@@ -15,89 +15,89 @@
 				"200x60_id": "data/ember_200x60.png",
 				"400x120_id": "data/ember_400x120.png"
 			},
-            "technology": "DLP",
```
Comment (ghost) [inline spark-print-data/printertypes.json:18]: a lot of whitespace issues in this makes it hard to check

**Comment by ghost** [comment]
Not sure we have a contributors agreement signed with you.

File: spark-print-data/printertypes.json (hunk 1 above)
Comment (ghost) [inline spark-print-data/printertypes.json:18]: Can you something about the white space in this return,

Comment (leyap) [inline spark-print-data/printertypes.json:18]: sorry, I can’t find white space

File: spark-print-data/printertypes.json [hunk 2]
```diff
@@ -656,10 +656,100 @@
...
+                "content": "application/gcode",
+                "thumbnail": "none",
+                "extension": "gcode",
+                "generates_supports": false
+            },
+            "supported_connections": [
+                {
+                    "type": "serial",
+                    "discovery": "serial",
+                    "protocol": "usb.serial",
```
Comment (ghost) [inline spark-print-data/printertypes.json:716]: Are you going to supply VID/PID for these connections https://github.com/spark3dp/print-manager/blob/master/spark-print-mgr/printers/drivers/driverConfig.json, however I can accept this as is. Let me know

**Comment by ghost** [comment]
I add to revert this pull request as it did not run.

**Pull request merged**